File: core/src/org/rexellentgames/dungeon/debug/DebugCommand.java
package org.rexellentgames.dungeon.debug;

import org.rexellentgames.dungeon.game.state.InGameState;
import org.rexellentgames.dungeon.physics.World;

public class DebugCommand extends ConsoleCommand {
	{
		shortName = "/d";
		name = "/debug";
		description = "Toggles physical debug";
	}

	@Override
	public void run(Console console, String[] args) {
		super.run(console, args);

		World.DRAW_DEBUG = !World.DRAW_DEBUG;
	}
}

File: core/src/org/rexellentgames/dungeon/debug/DebugCommand.kt
package org.rexellentgames.dungeon.debug

import org.rexellentgames.dungeon.physics.World

class DebugCommand : ConsoleCommand("/debug", "/d", "Toggles physical debug") {
    override fun run(console: Console, args: Array<String>) {
        super.run(console, args)

        World.DRAW_DEBUG = !World.DRAW_DEBUG
    }
}